### PR TITLE
Initial EMIS backend deployment

### PR DESCRIPTION
**Pull request by bloodearnest**
- we don't have api.github.com access, so switch from ssh-import-id to
 curl.
 - manage a /srv/jobserver/secret dir for the certs
 - add EMIS variables to the secrets-template.env
 - fix a bug in the sourcing of multiple files

### CODE REVIEW

File: scripts/update-ssh.sh [hunk 1]
```diff
@@ -19,7 +19,7 @@ if test -f "$cache" -a -n "${TEST}"; then
     cat "$cache" >> "$tmp"
 else
     # add current gh keys into $tmp
-    ssh-import-id "gh:$user" --output "$tmp"
+    curl -s "https://github.com/$user.keys" -o "$tmp"
```
Comment (madwort) [inline scripts/update-ssh.sh:22]: if the user already has an explicit key from this repo, will this do the right thing (i.e. append to the file)?

Comment (bloodearnest) [reply scripts/update-ssh.sh]: Thanks, fixed with `... >> $tmp`

File: jobrunner/bashrc [hunk 2]
```diff
@@ -1,7 +1,9 @@
 # load config into this shell so we can just run stuff
 set -a
 # shellcheck disable=SC1090
-. /srv/jobrunner/environ/*.env
+for f in /srv/jobrunner/environ/*.env; do
+    . $f
```
Comment (madwort) [inline jobrunner/bashrc:5]: should this be `. "$f"` or it doesn't matter? (nb. how come this doesn't trigger https://github.com/koalaman/shellcheck/wiki/sc2086 ?)

File: scripts/jobrunner.sh [hunk 3]
```diff
@@ -80,6 +82,9 @@ test -f "$playbook" && ln -sf "$playbook" ~jobrunner/playbook.md
 
 # ensure file ownership and permissions
 chown -R jobrunner:jobrunner /srv/jobrunner
+chmod 0600 $secrets_env
+chmod 0700 $DIR/secret
+find $DIR/secret -type f -exec chmod 0600 {} \;
```
Comment (madwort) [inline scripts/jobrunner.sh:87]: should this be `"$DIR"/secret ...`?

Comment (madwort) [reply scripts/jobrunner.sh:87]: hmm maybe not - it's not done everywhere, so...

**Review by madwort — APPROVED** [review]
Some questions about fragility of quoting, but this is bash, and I think it's fine for now

File: scripts/jobrunner.sh (hunk 3 above)
Comment (bloodearnest) [reply scripts/jobrunner.sh:87]: It should but why did the lint checks pass if so?

I authored these changes on the EMIS VM, where we currently can't install stuff (like shellcheck)

Comment (madwort) [reply scripts/jobrunner.sh:87]: narrator: it was because the linter spotted that the var `DIR` did not have any spaces, so did not need to be quoted

**Review by madwort — APPROVED** [review]
👍

**Pull request merged**